src/HABApp/openhab/definitions/websockets/item_value_types.py

```diff
@@ -411,7 +411,7 @@ def from_value(cls, value: Any) -> Self | None:
 # https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/RawType.java
 class RawTypeModel(ItemValueBase):
     type: Literal['Raw']
-    value: str = Field(pattern=r'^data:image/[a-z_-]+;base64,[A-Za-z0-9+/]*={0,2}$')
+    value: str = Field(pattern=r'^data:image/[a-z_\-+.]+;base64,[A-Za-z0-9+/]*={0,2}$')
```

Comment:
Is `\-` meant to escape `-` or should it also match? If so maybe move the `-` to the end of the regex?